examples/spring-boot-webflux/src/main/resources/config/application.yml

```diff
@@ -0,0 +1,11 @@
+spring.profiles.active: local
+server:
+  port: 8443
+  ssl:
+    enabled: true # Use self-signed certificate because no key-store configuration is specified.
+
+# Additional server ports specified with Armeria configuration.
+armeria:
+  ports:
+  - port: 8443    # Port 8443 will serve both HTTPS and HTTP.
```

Comment:
This is interesting. Is this an expected behavior to serve both HTTP and HTTPS on 8443 in this case? I guess a user may expect HTTPS only?